test/saga.test.js

```diff
@@ -1,12 +1,12 @@
-/*globals Promise:true*/
-global.fetch = require('jest-fetch-mock');
+import { takeEvery } from 'redux-saga/effects';
+import { expectSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+import { throwError } from 'redux-saga-test-plan/providers';
 
-import { call, put, takeEvery } from 'redux-saga/effects';
-
-import { FETCH, get } from '../src/actions';
+import { del, FETCH, get, post } from '../src/actions';
 import saga, { fetchSaga } from '../src/saga';
 
-test('the main saga is just `takeEvery`', () => {
+test('the main saga watcher is just `takeEvery`', () => {
```

Comment:
can we fix the test too with `redux-saga-test-plan`

Reply:
i think so, but as is, the test is exactly what the name describes. IMO this is a case where we do want to explicitly tests the generator itself, since we're being super-specific about what we want the function to do.

Reply:
okay that makes sense